src/ceph-detect-init/CMakeLists.txt

```diff
@@ -1,8 +1,14 @@
 set(CEPH_DETECT_INIT_VIRTUALENV ${CEPH_BUILD_VIRTUALENV}/ceph-detect-init-virtualenv)
 
+if(WITH_PYTHON3)
```

Comment:
strictly speaking, it's not right. we build python bindings for py3 if `WITH_PYTHON3=ON`. but ceph-detect-init does not use any of our python bindings.